### ISSUE TITLE
Not being able to add treatments

### ISSUE THREAD

**Issue by Foxandxss** [NightscoutFoundation/xDrip#1465]
Hello, I have a nightscout follower xDrip that I updated from a old version to:

Version:a91b660-2020.10.1 Code: 2010011129

When I click the treatment icon (the one that looks like a color picker) and I add the units, the green checkmark that used to confirm the treatment is not working. It does absolutely nothing for me.

I can see the "5 units" but I can't move from there.

Thank you.

**Comment by tolot27** [comment]
Please can you post a screenshot? Also did you enter just the insulin data? Sometimes on one of the other pages there is a wrong or incomplete data given. In that case, you cannot confirm the treatment.

Please be aware of #735.

**Comment by Foxandxss** [comment]
I was going to upload an screenshot but before doing that I did check that I didn't have anything in the other tabs (time, carbs, ...) so I checked all of them and now it does work. That is very weird.

xDrip has been asking me to update for ages so one morning (it seems to ask in the mornings) asked for update, said yes, installed it and then I was going to add a bolus and it didn't work and didn't work for days (closing the app entirely and things like that). Now just moving through the options fixed it.

Sorry for distractions. Let me close this.

Thank you sir.
